### PR TITLE
features: Html and Css files added

### PR DESCRIPTION
**Pull request by Adesina-Abiodun**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    background-color: brown;
+    border-color: brown;
+  }
+  
+  .bottom-line {
+    margin-top: 25px;
+  }
+  
+  h1, h2 {
+    font-family: Impact;
```
Comment (Adesina-Abiodun) [inline styles.css:51]: Please consider adding a generic font-family e.g. san-serif, serif, etc. after your unique font-family for accessibility.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# Code-review-template
```
Comment (Adesina-Abiodun) [inline README.md:1]: Please consider using the Microverse READEME.md template found [here](https://github.com/microverseinc/readme-template#about-projectl) in order to have more professional documentation 📝

_A Professional Readme serves as a guide that explains the purpose of your project, provides instructions for installation, offers guidance on how to use it, and includes information on how to contribute to the project._

**Review by Adesina-Abiodun — COMMENTED** [review]
**Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!**

•	Ensure to have good PR description 
•	Give a more meaningful commit messages 
•	Have good project documentation

File: README.md (hunk 2 above)
Comment (usorfaitheloho) [reply README.md:1]: Great job adding an inline comment and including the resource where the template can be found, please I'd suggest you give an elaborate reason why the student needs to document his project 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Adesina-Abiodun  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Used Inline comments ✔️
> * Gave a polite suggestion ✔️
> * Gave a reason why the change should be done✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Adesina-Abiodun — COMMENTED** [review]
Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Adesina-Abiodun , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You met the requirement adding at least 2 inline comments. ✔
- Well done implementing the requestes changes. ✔
- Your comments sound polite and very descriptive now. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**